tools/babel-config.js

```diff
@@ -0,0 +1,21 @@
+// This is the same set of transformations that we will enable in a future
+// official React package
+module.exports = {
+  sourceMaps: "inline",
+  compact: false,
+  whitelist: [
```

Comment:
Or at least share it in the hacky way that we share package-version-parser.  I don't think isopacket will work well because this is part of the build tool.